agent/spec/lib/kontena/launchers/etcd_launcher_spec.rb

```diff
@@ -94,14 +98,14 @@
       allow(container).to receive(:running?).and_return(true)
       allow(container).to receive(:info).and_return({'Config' => {'Image' => 'etcd'}})
       expect(subject.wrapped_object).to receive(:add_dns)
-      node_info = {
+      node = Node.new(
```

Comment:
Let's make conflicts great again... I think we should merge #1323 first and then we can see how to proceed with this.